packages/orbit-components/src/SegmentedSwitch/index.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import getSpacingToken from "../common/getSpacingToken";
+import FormLabel from "../FormLabel";
+import defaultTheme from "../defaultTheme";
+import Stack from "../Stack";
+import SwitchSegment from "./SwitchSegment";
+import ErrorFormTooltip from "../ErrorFormTooltip";
+import useErrorTooltip from "../ErrorFormTooltip/hooks/useErrorTooltip";
+
+import type { Props } from ".";
+
+const StyledWrapper = styled.label`
+  ${({ theme, $maxWidth }) => css`
+    display: flex;
+    width: 100%;
+    position: relative;
+    flex-direction: column;
+    max-width: ${$maxWidth};
+    margin-bottom: ${getSpacingToken};
+    gap: ${theme.orbit.spaceXXXSmall};
+  `};
+`;
+
+// $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
+StyledWrapper.defaultProps = {
+  theme: defaultTheme,
+};
+
+const SegmentedSwitch = ({
+  options,
+  dataTest,
+  onChange,
+  spaceAfter,
+  onFocus,
+  maxWidth,
+  help,
+  error,
+  label,
+}: Props): React.Node => {
+  const {
+    tooltipShown,
+    tooltipShownHover,
+    setTooltipShown,
+    setTooltipShownHover,
+    labelRef,
+    handleFocus,
+  } = useErrorTooltip({ onFocus });
```

Comment:
I believe that doing this is causing the tooltip to be triggered when we select an option. Is that expected? I had the feeling we would only want the tooltip to be manually triggered **or** on hover. How can we know for sure when is the tooltip expected to be shown?